mediator/med_phases_history_mod.F90

```diff
@@ -74,6 +76,7 @@ module med_phases_history_mod
   ! Time averaging history files
   ! ----------------------------
   type, public :: avgfile_type
+     type(file_desc_t) :: io_file
```

Comment:
Great implementation. However, this is very picky - but could you please keep the indentation for :: the same.